### ISSUE TITLE
[BUG] Admiralty file input failing Axe-core accessibility tests

### ISSUE THREAD

**Issue by LukeTaylor34** [UKHO/admiralty-design-system#116]
**Describe the bug**
When running Axe-core automated tests on a page with the file input component. The tests will fail due to the file input having an Aria-hidden="true" attribute while the input has focusable content.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to File input component page
2. Inspect component 
3. The input has an Aria-hidden="true"

![image](https://github.com/UKHO/admiralty-design-system/assets/97240814/da8fa791-0d35-4cfc-9af1-1a84b68a978a)

Also raised by Agastya
